src/org/opensolaris/opengrok/history/BazaarRepository.java

```diff
@@ -295,57 +263,23 @@ boolean hasFileBasedTags() {
      * @param directory Directory where we list tags
      */
     @Override
-    protected void buildTagList(File directory) {
+    protected void buildTagList(File directory, boolean interactive) {
```

Comment:
Wondering if new `interactive` should be used in this method.

Reply:
ditto